tests/unit/cellguide_pipeline/test_canonical_marker_genes.py

```diff
@@ -0,0 +1,65 @@
+import json
+import unittest
+
+from backend.cellguide.pipeline.canonical_marker_genes.canonical_markers import CanonicalMarkerGenesCompiler
+from backend.cellguide.pipeline.utils import convert_dataclass_to_dict_and_strip_nones
+from tests.unit.backend.wmg.fixtures.test_snapshot import (
+    load_realistic_test_snapshot,
+)
+
+TEST_SNAPSHOT = "realistic-test-snapshot"
+
+
+def compare_dicts(dict1, dict2):
```

Comment:
After merging your PR, once I update from main I will remove this and update to import from the utility module you created